### PR TITLE
divider component

### PR DESCRIPTION
**Pull request by liorMar**
implemented divider component with style override capabilities

### CODE REVIEW

File: stories/Divider.story.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  component: Divider,
+  source: DividerSource,
+  componentPath: '../src/components/Divider/Divider.tsx',
+  componentProps: () => ({
+    'data-hook': 'storybook-Divider',
+    // children: 'Some text',
+    // typography: TYPOGRAPHY.runningText,
+  }),
+  exampleProps: {
+    // typography: Object.keys(TYPOGRAPHY).map(key => TYPOGRAPHY[key]),
```
Comment (shlomitc) [inline stories/Divider.story.tsx]:
```suggestion

```

File: stories/Divider.story.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+export default {
+  category: 'Components',
+  storyName: 'Divider',
+  component: Divider,
+  source: DividerSource,
+  componentPath: '../src/components/Divider/Divider.tsx',
+  componentProps: () => ({
+    'data-hook': 'storybook-Divider',
+    // children: 'Some text',
```
Comment (shlomitc) [inline stories/Divider.story.tsx]:
```suggestion

```

File: stories/Divider.story.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    // typography: TYPOGRAPHY.runningText,
```
Comment (shlomitc) [inline stories/Divider.story.tsx]:
```suggestion

```

File: stories/Divider.story.tsx (hunk 1 above)
Comment (liorMar) [reply stories/Divider.story.tsx]: @shlomitc done

File: stories/Divider.story.tsx (hunk 3 above)
Comment (liorMar) [reply stories/Divider.story.tsx]: @shlomitc done

**Pull request merged**